From 9f27163a2b4704f015827516cc86aa4b54729527 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@nowhere.nt>
Date: Mon, 9 Nov 2020 22:40:42 +0100
Subject: [PATCH 1/2] qt/app.h: Add missing #include <wx/scopedptr.h>.

---
 include/wx/qt/app.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/wx/qt/app.h b/include/wx/qt/app.h
index b9f21141aa..fc5d8a8f6b 100644
--- a/include/wx/qt/app.h
+++ b/include/wx/qt/app.h
@@ -10,6 +10,7 @@
 #define _WX_QT_APP_H_
 
 #include <wx/scopedarray.h>
+#include <wx/scopedptr.h>
 
 class QApplication;
 class WXDLLIMPEXP_CORE wxApp : public wxAppBase
-- 
2.26.2

